inst/tutorials/introduction/introduction.Rmd

```diff
@@ -0,0 +1,265 @@
+     question_text("What is the default mosquito population size?",
+                   answer("1000", correct = TRUE), allow_retry = T),
+     question_numeric("What is the default initial susceptible human state proportion (to 2 dp)?",
+                      answer("0.42", correct = TRUE), allow_retry = T),
+     question("How much immunity to malaria does the initial population have?",
+              answer("0", correct = TRUE),
+              answer("2.4"),
+              answer("20"),
+              answer("53"), allow_retry = T,
+              correct = "That's right! The population begins with no immunity. This isn't realistic for most settings, and we will describe how to adjust this during the tutorial."
+     )
+)
+
+```
+
+
+Any of these parameters can be adjusted within the `get_parameters()` function using the overrides argument. For example, we can increase the human population size (`human_population`) like this:
+
+```{r parameters_override, exercise=TRUE}
+get_parameters(overrides = list(human_population = 1000))
+```
+
+Most of the parameters have been carefully calibrated so will not need adjusting manually. One exception is the human population size.
+
+Try changing the population size to 250,000 and check that this has updated in the parameter list.
+
+```{r parameters_override_check, exercise=TRUE}
+
+```
+
+
+We generally store a parameter set in an object for future use like so:
+
+```{r parameters_store, exercise=TRUE}
+parameters <- get_parameters()
+```
+
+By so doing, we can also adjust individual parameters manually outside of the `get_parameters()` function.
+
+```{r parameters_alternative, exercise=TRUE}
+parameters <- get_parameters()
+parameters$human_population <- 1000
+parameters$human_population
+```
+
+
+`malariasimulation` also contains various helper functions and in-built parameters to help you design your parameter set. We will not cover these in detail here, but they include setting:
+
+- population demographies `set_demography()`,
+- mosquito species and proportions `set_species()`, with in-built species specific parameter sets: An. arabiensis `arab_params()`, An. funestus `fun_params()`, An. gambiae `gamb_params()` and An. stephensi `steph_params()`,
+- drug interventions including MDA and SMC `set_drugs()`, `set_clinical_treatment()`, `set_mda()`, `set_smc()`,
+- bednet and IRS vector control `set_bednets()`, `set_spraying()`,
+- pre-erythrocytic and transmission blocking vaccines `set_mass_pev()`, `set_pev_epi()`, `set_tbv()`.
+
+Please see the package vignettes for more details on how to use these, or ask a demonstrator!
+
+
+
+
+## 2: Set equilibrium
+
+Do you remember seeing default parameters for the initial human disease state proportions and human immunity levels? These may not be applicable to the scenario you are trying to simulate (e.g., particularly if you are starting a simulation with no immunity to malaria in a high-transmission malaria-endemic area!).
```

Comment:
Not sure how clear this intro would be for a new-comer. Does the following read with more clarity:

```
To change the intensity of transmission in the model, follow the steps below:

    1. Understand Default Parameters: The default parameters for the initial human disease state proportions and human immunity levels may not suit the scenario you are simulating. For instance, if you are starting a simulation with no immunity to malaria in a high-transmission area, the default settings won't be appropriate.

    2. Use the set_equilibrium() Function: This function helps set the initial transmission level and calculates an equilibrium based on that transmissibility level and your chosen model parameters.

    3. Define the Initial EIR: In set_equilibrium(), you will define the initial EIR (entomological inoculation rate), The Entomological Inoculation Rate (EIR) is a key metric used in malaria epidemiology. It is defined as the number of infectious bites received by a person per unit of time. Adjusting the initial EIR allows you to reflect the disease prevalence in the region you are studying.
```